src/QMCWaveFunctions/detail/SYCL/sycl_determinant_helper.hpp

```diff
@@ -31,5 +30,14 @@ sycl::event applyW_stageV_sycl(sycl::queue& aq,
                                const int ndelay,
                                T* V_gpu,
                                const T* Ainv);
-}
+
+template<typename T, typename TMAT, typename INDEX>
+std::complex<T> computeLogDet_sycl(sycl::queue& aq,
+                                   int n,
+                                   int lda,
+                                   const TMAT* a,
+                                   const INDEX* pivot,
```

Comment:
Likely we should just hard-code int64_t

Reply:
I don't feel it is better than what CUDA/HIP does due to its blocking nature. CUDA code path just transfer the pivot and diagonal terms to host asynchronously.